### DISCUSSION TITLE
How to uninstall or remove a Quarto version?

### DISCUSSION THREAD

**Discussion by camilogarciabotero** [dpastoor/qvm#12] [Q&A] [answered]
Hi Devin,

Thanks for working on this project. I was wondering if there is a way to uninstall or remove a Quarto version.

Thanks in advance.

**Comment by dpastoor** [comment, marked as answer]
edit: implemented as `qvm rm ...`

I definitely need to add this ASAP! It's next on my to-do and should be able to get to it in the next couple days

**Reply by dpastoor** [reply]
I have implemented this https://github.com/dpastoor/qvm/pull/13 and am currently building a v0.1.0 release that will include this as `qvm rm <versions...>`

**Reply by camilogarciabotero** [reply]
Thank you so much!
